### PR TITLE
adding liquidity add/remove tests

### PR DESCRIPTION
**Pull request by MariuszSzpyt**
Tests are passing:
https://mangata-finance.testmo.net/automation/runs/view/256

- Upgraded chrome driver and selenoid chrome to 112.0.0
- Added Liquidity adding and removing tests
- Liquidity adding test ends up on toggling & validating pool tab  - as we have already providing pool action tested in separate test
- Minor cleanups and fixes

### CODE REVIEW

File: test/ui/ui-main.poolsLiquidity.addRemove.test.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    testAssetId = await createAssetIfMissing(sudo, testAssetName);
+    await createPoolIfMissing(
+      sudo,
+      "1800000000000000000000000000000000",
+      MGA_ASSET_ID,
+      testAssetId
+    );
+
+    await Sudo.batchAsSudoFinalized(
+      Assets.mintToken(new BN(7), testUser1), // transferAll test
```
Comment (goncer) [inline test/ui/ui-main.poolsLiquidity.addRemove.test.ts:79]: 7 ? why this one?

File: utils/frontend/utils/Driver.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const DriverBuilder = (function () {
 
     let caps: Capabilities = new Capabilities();
     caps = Capabilities.chrome();
-    caps.set("version", "110.0");
+    caps.set("version", "112.0");
```
Comment (goncer) [inline utils/frontend/utils/Driver.ts:30]: perhaps the version should come from an env. 
looks like tedious tasks updating chomeDriver, isn't it?

**Review by goncer — APPROVED** [review]

File: test/ui/ui-main.poolsLiquidity.addRemove.test.ts (hunk 1 above)
Comment (MariuszSzpyt) [reply test/ui/ui-main.poolsLiquidity.addRemove.test.ts:79]: ha :D not needed, just an oversight, fixing

File: utils/frontend/utils/Driver.ts (hunk 2 above)
Comment (MariuszSzpyt) [reply utils/frontend/utils/Driver.ts:30]: true! will improve in next PR

**Pull request merged**